apps/omstillingsstoenad-ui/cypress/e2e/avbryt_soeknad.cy.js

```diff
@@ -3,7 +3,7 @@ import { basePath, getById } from "../util/cy-functions";
 describe("Skal avbryte en soeknad",() => {
     const startSoeknad = () => {
         // Gå til søknad med eksisterende kladd
-        cy.intercept("GET", `${basePath}/api/person/innlogget`, {fixture: "testbruker"}).as("hentInnloggetPerson");
+        cy.intercept("GET", `${basePath}/api/person/innlogget?soeknadType=BARNEPENSJON`, {fixture: "testbruker"}).as("hentInnloggetPerson");
```

Comment:
Spiller vel ingen stor rolle siden det er en test, men kanskje ryddig å bruke rett type likevel. Samme gjelder de andre cypress-testene for OMS.